ts/Stock/RangeSelector/RangeSelector.ts

```diff
@@ -1310,36 +1319,61 @@ class RangeSelector {
 
         const dropdown = this.dropdown = createElement('select', void 0, {
             position: 'absolute',
-            width: '1px',
-            height: '1px',
             padding: 0,
             border: 0,
-            top: '-9999em',
             cursor: 'pointer',
             opacity: 0.0001
         }, this.div) as HTMLSelectElement;
 
+        // Create a label for dropdown select element
+        const userButtonTheme = chart.userOptions.rangeSelector?.buttonTheme;
+        this.dropdownLabel = renderer.button(
+            '',
+            0,
+            0,
+            (): void => {},
+            merge(buttonTheme, {
+                'stroke-width': pick(buttonTheme['stroke-width'], 0),
+                width: 'auto',
+                paddingLeft: pick(
+                    options.buttonTheme.paddingLeft,
+                    userButtonTheme?.padding,
+                    8
+                ),
+                paddingRight: pick(
+                    options.buttonTheme.paddingRight,
+                    userButtonTheme?.padding,
+                    8
+                )
+            }),
+            states && states.hover,
+            states && states.select,
+            states && states.disabled
+        )
+            .add(this.buttonGroup)
+            .hide();
```

Comment:
Swap this 2 lines. (Theoretically, painting of the screen is done after the whole JS is performed, but why risk it, to make the dropdown label displayed for a split ms?)